filebeat/input/filestream/config.go

```diff
@@ -41,6 +41,7 @@ type config struct {
 	HarvesterLimit uint32                  `config:"harvester_limit" validate:"min=0"`
 	IgnoreOlder    time.Duration           `config:"ignore_older"`
 	IgnoreInactive ignoreInactiveType      `config:"ignore_inactive"`
+	Rotation       *common.ConfigNamespace `config:"rotation"`
```

Comment:
we might want to find another name here. `rotation` suggests that the input is capable of rotating the file by itself.